react/week2/todo-list/src/Components/data.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I would move this file outside of the Components folder.

Reply:
Thanks Svetlana for your feedback..
Implemented the feedback